<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AssignUser</class>
 <widget class="QDialog" name="AssignUser">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>481</width>
    <height>450</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: #1B2131;</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>471</width>
     <height>351</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox{
background-color: #1B2131;
border:#1B2131;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>35</x>
      <y>75</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(208, 208, 208);</string>
    </property>
    <property name="text">
     <string>Project : </string>
    </property>
   </widget>
   <widget class="QLabel" name="lblProject">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>75</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;</string>
    </property>
    <property name="text">
     <string>Project Code </string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>160</y>
      <width>341</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(208, 208, 208);</string>
    </property>
    <property name="text">
     <string>Select the employee to assign it to the project</string>
    </property>
   </widget>
   <widget class="QComboBox" name="cbStaff">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>301</width>
      <height>35</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#cbStaff{
background-color: #151925;
border: #1B2131;
color: #fff;
padding-left:7px;
}</string>
    </property>
    <property name="placeholderText">
     <string>Select the employee number</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblProjectName">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>75</y>
      <width>281</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color:#fff;</string>
    </property>
    <property name="text">
     <string>Project name</string>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="btnAssign">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>360</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:0px;
color: #EE5F8E;
border: 2px solid #EE5F8E;
background-color: transparent;</string>
   </property>
   <property name="text">
    <string>Assign</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
